agents/labjack/Dockerfile

```diff
@@ -9,6 +9,18 @@ WORKDIR /app/socs/agents/labjack/
 
 ## Copy this agent into the app/agents directory
 COPY . .
+RUN pip3 install -r requirements.txt
+
+# Install ljm module from labjack
```

Comment:
Can we add a comment to link to the labjack/ljm image on Dockerhub for reference? It's this one, yeah?
https://hub.docker.com/r/labjack/ljm

That gives context for the Dockerfile additions and provides a good reference for if we need future updates that they might implement.

Reply:
Sure, just did it in the latest commit